variables.tf

```
@@ -25,7 +25,7 @@ variable "admin_email" {
 }
 
 variable "cluster_version" {
-  default = "1.17"
+  default = "1.19"
```

We should make this 1.21 since 1.19 will be EOL in 5 months. https://github.com/astronomer/ops-stuff/blob/main/docs/k8s-version-table.md
```suggestion
  default = "1.21"
```